applications/luci-app-v2raya/htdocs/luci-static/resources/view/v2raya/config.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+'require form';
+'require poll';
+'require rpc';
+'require uci';
+'require view';
+
+var callServiceList = rpc.declare({
+	object: 'service',
+	method: 'list',
+	params: ['name'],
+	expect: { '': {} }
+});
+
+function getServiceStatus() {
+	return L.resolveDefault(callServiceList('v2raya'), {}).then(function (res) {
+		var isRunning = false;
+		try {
+			isRunning = res['v2raya']['instances']['v2raya']['running'];
+		} catch (e) { }
+		return isRunning;
+	});
+}
+
+function renderStatus(isRunning, port) {
+	var spanTemp = '<span style="color:%s"><strong>%s %s</strong></span>';
+	var renderHTML;
+	if (isRunning) {
+		var button = String.format('&#160;<a class="btn cbi-button" href="http://%s:%s" target="_blank" rel="noreferrer noopener">%s</a>',
+			window.location.hostname, port, _('Open Web Interface'));
+		renderHTML = spanTemp.format('green', _('v2rayA'), _('RUNNING')) + button;
+	} else {
+		renderHTML = spanTemp.format('red', _('v2rayA'), _('NOT RUNNING'));
+	}
+
+	return renderHTML;
+}
+
+return view.extend({
+	load: function() {
+		return Promise.all([
+			uci.load('v2raya')
+		]);
+	},
+
+	render: function(data) {
+		var m, s, o;
+		var webport = (uci.get(data[0], 'config', 'address') || '0.0.0.0:2017').split(':').slice(-1)[0];
+
+		m = new form.Map('v2raya', _('v2rayA'),
+			_('v2rayA is a V2Ray Linux client supporting global transparent proxy, compatible with SS, SSR, Trojan(trojan-go), PingTunnel protocols.'));
+
+		s = m.section(form.TypedSection);
+		s.anonymous = true;
+		s.render = function () {
+			poll.add(function () {
+				return L.resolveDefault(getServiceStatus()).then(function (res) {
+					var view = document.getElementById('service_status');
+					view.innerHTML = renderStatus(res, webport);
+				});
+			});
+
+			return E('div', { class: 'cbi-section', id: 'status_bar' }, [
+					E('p', { id: 'service_status' }, _('Collecting data…'))
+			]);
+		}
+
+		s = m.section(form.NamedSection, 'config', 'v2raya');
+
+		o = s.option(form.Flag, 'enabled', _('Enable'));
+		o.rmempty = false;
+
+		o = s.option(form.Value, 'address', _('Listening address'));
+		o.datatype = 'ipaddrport(1)';
+		o.placeholder = '0.0.0.0:2017';
+
+		o = s.option(form.ListValue, 'ipv6_support', _('IPv6 support'),
+			_('Make sure your IPv6 network works fine before you turn it on.'));
```

Comment:
Shorten to:
```js
			_('Requires working IPv6 connectivity.'));
```

But does this mean to the Internet? In which case:

```js
			_('Requires IPv6 Internet connectivity.'));
```

Reply:
It just requies IPv6 support to be enabled on kernel level.

Reply:
Then modify it to:
```js
			_('Requires working IPv6 connectivity.'));
```

Reply:
Done ;)